manifest.json

```diff
@@ -6,10 +6,9 @@
     "48": "icons/48-enabled.png",
    "128": "icons/128-enabled.png" 
   },
-  "manifest_version": 2,
+  "manifest_version": 3,
   "options_page": "src/options/index.html",
-  "content_security_policy": "script-src 'self' 'unsafe-eval'; object-src 'self'",
```

Comment:
No longer needed now that vue is not used (and therefore `unsafe-eval` is no longer used)